google-scripts/package.json

```diff
@@ -11,5 +11,6 @@
     "@types/jest": "^26.0.24",
     "jest": "^27.0.6",
     "ts-jest": "^27.0.4"
-  }
+  },
+  "dependencies": {}
```

Comment:
Might need to remove this? Not sure why it's here 😅

Reply:
it shouldn't harm anything